gravatar-quickeditor/src/main/res/values/strings.xml

```diff
@@ -19,6 +19,7 @@
     <string name="gravatar_qe_photo_library_icon_description">Photo library</string>
     <string name="gravatar_qe_capture_photo_icon_description">Capture Photo</string>
     <string name="gravatar_qe_avatar_selection_error">Ooops, there was an error selecting the avatar.</string>
+    <string name="gravatar_qe_avatar_delete_avatar_error">Error deleting the image.</string>
```

Comment:
Do we get any meaningful errors from the server that we could use? Not necessarily directly, but to make this message less generic?

Reply:
Yeah, we should. But we need that one to be handled more globally so that we don't have to repeat this on each request. So likely a separate issue anyway.

Reply:
Agree. #453